<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(0, 0, 0);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>816</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">backgrund-color:rgb(214, 214, 214);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>20</y>
        <width>271</width>
        <height>271</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white</string>
      </property>
      <column>
       <property name="text">
        <string>메뉴</string>
       </property>
       <property name="font">
        <font>
         <family>더잠실 3 Regular</family>
         <pointsize>9</pointsize>
         <stylestrategy>PreferDefault</stylestrategy>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignLeading|AlignVCenter</set>
       </property>
       <property name="foreground">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </property>
      </column>
      <column>
       <property name="text">
        <string>수량</string>
       </property>
       <property name="font">
        <font>
         <family>더잠실 3 Regular</family>
         <pointsize>9</pointsize>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignLeading|AlignVCenter</set>
       </property>
       <property name="foreground">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </property>
      </column>
      <column>
       <property name="text">
        <string>금액</string>
       </property>
       <property name="font">
        <font>
         <family>더잠실 3 Regular</family>
         <pointsize>9</pointsize>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignLeading|AlignVCenter</set>
       </property>
       <property name="foreground">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>290</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>더잠실 3 Regular</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:black;
background-color:rgb(214, 214, 214)</string>
      </property>
      <property name="text">
       <string>합계                          N원</string>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>481</width>
        <height>341</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="tabletTracking">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(214, 214, 214)</string>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>479</width>
         <height>339</height>
        </rect>
       </property>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>10</y>
          <width>181</width>
          <height>131</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>더잠실 3 Regular</family>
          <pointsize>26</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:black;
background-color:rgb(214, 214, 214)</string>
        </property>
        <property name="text">
         <string>추천메뉴2</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>181</width>
          <height>131</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>더잠실 3 Regular</family>
          <pointsize>26</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:black;
background-color:rgb(214, 214, 214)</string>
        </property>
        <property name="text">
         <string>추천메뉴1</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>10</y>
          <width>181</width>
          <height>131</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>더잠실 3 Regular</family>
          <pointsize>26</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:black;
background-color:rgb(214, 214, 214)</string>
        </property>
        <property name="text">
         <string>추천메뉴3</string>
        </property>
       </widget>
       <widget class="QScrollBar" name="horizontalScrollBar">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>320</y>
          <width>481</width>
          <height>20</height>
         </rect>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>170</y>
          <width>181</width>
          <height>131</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>더잠실 3 Regular</family>
          <pointsize>26</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:black;
background-color:rgb(214, 214, 214)</string>
        </property>
        <property name="text">
         <string>추천메뉴4</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_8">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>170</y>
          <width>181</width>
          <height>131</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>더잠실 3 Regular</family>
          <pointsize>26</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:black;
background-color:rgb(214, 214, 214)</string>
        </property>
        <property name="text">
         <string>추천메뉴5</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_9">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>170</y>
          <width>181</width>
          <height>131</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>더잠실 3 Regular</family>
          <pointsize>26</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:black;
background-color:rgb(214, 214, 214)</string>
        </property>
        <property name="text">
         <string>추천메뉴6</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_18">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>330</y>
        <width>271</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>더잠실 3 Regular</family>
        <pointsize>26</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:red;
color:white</string>
      </property>
      <property name="text">
       <string>결제하기</string>
      </property>
     </widget>
     <widget class="QListView" name="listView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>781</width>
        <height>391</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(214, 214, 214)</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>60</y>
        <width>271</width>
        <height>192</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white</string>
      </property>
      <item>
       <property name="text">
        <string>hhbhbbh            3               3000</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>asdfsadfsdf</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>sdafsdfas</string>
       </property>
      </item>
     </widget>
     <zorder>listView</zorder>
     <zorder>tableWidget</zorder>
     <zorder>label_2</zorder>
     <zorder>scrollArea</zorder>
     <zorder>pushButton_18</zorder>
     <zorder>listWidget</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QListView" name="listView_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>791</width>
        <height>391</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(214, 214, 214)</string>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollArea_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>481</width>
        <height>341</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(214, 214, 214)</string>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>479</width>
         <height>339</height>
        </rect>
       </property>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>10</y>
          <width>181</width>
          <height>131</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>더잠실 3 Regular</family>
          <pointsize>26</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:black;
background-color:rgb(214, 214, 214)</string>
        </property>
        <property name="text">
         <string>세트메뉴2</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>181</width>
          <height>131</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>더잠실 3 Regular</family>
          <pointsize>26</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:black;
background-color:rgb(214, 214, 214)</string>
        </property>
        <property name="text">
         <string>세트메뉴1</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_6">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>10</y>
          <width>181</width>
          <height>131</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>더잠실 3 Regular</family>
          <pointsize>26</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:black;
background-color:rgb(214, 214, 214)</string>
        </property>
        <property name="text">
         <string>세트메뉴3</string>
        </property>
       </widget>
       <widget class="QScrollBar" name="horizontalScrollBar_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>319</y>
          <width>481</width>
          <height>20</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_10">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>170</y>
          <width>181</width>
          <height>131</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>더잠실 3 Regular</family>
          <pointsize>26</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:black;
background-color:rgb(214, 214, 214)</string>
        </property>
        <property name="text">
         <string>세트메뉴</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_11">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>170</y>
          <width>181</width>
          <height>131</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>더잠실 3 Regular</family>
          <pointsize>26</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:black;
background-color:rgb(214, 214, 214)</string>
        </property>
        <property name="text">
         <string>세트메뉴1</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_12">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>170</y>
          <width>181</width>
          <height>131</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>더잠실 3 Regular</family>
          <pointsize>26</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:black;
background-color:rgb(214, 214, 214)</string>
        </property>
        <property name="text">
         <string>세트메뉴1</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QTableWidget" name="tableWidget_2">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>20</y>
        <width>271</width>
        <height>271</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white</string>
      </property>
      <column>
       <property name="text">
        <string>메뉴</string>
       </property>
       <property name="font">
        <font>
         <family>더잠실 3 Regular</family>
         <pointsize>12</pointsize>
         <stylestrategy>PreferDefault</stylestrategy>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignLeading|AlignVCenter</set>
       </property>
       <property name="foreground">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </property>
      </column>
      <column>
       <property name="text">
        <string>수량</string>
       </property>
       <property name="font">
        <font>
         <family>더잠실 3 Regular</family>
         <pointsize>12</pointsize>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignLeading|AlignVCenter</set>
       </property>
       <property name="foreground">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </property>
      </column>
      <column>
       <property name="text">
        <string>금액</string>
       </property>
       <property name="font">
        <font>
         <family>더잠실 3 Regular</family>
         <pointsize>12</pointsize>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignLeading|AlignVCenter</set>
       </property>
       <property name="foreground">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </property>
      </column>
      <column>
       <property name="text">
        <string>새 행</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>새 행</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>290</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>더잠실 3 Regular</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:black;
background-color:rgb(214, 214, 214)</string>
      </property>
      <property name="text">
       <string>합계                          N원</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_19">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>330</y>
        <width>271</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>더잠실 3 Regular</family>
        <pointsize>26</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:red;
color:white</string>
      </property>
      <property name="text">
       <string>결제하기</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>0</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>더잠실 3 Regular</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white</string>
    </property>
    <property name="text">
     <string>N번 테이블 주문</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
